### ISSUE TITLE
SDK37 - Unable to resolve module `shortid` from StackRouter

### ISSUE THREAD

**Issue by xn3cr0nx** [expo/expo#7578]
### Environment

Expo CLI 3.17.5 environment info:
    System:
      OS: Linux 5.3 Ubuntu 19.10 (Eoan Ermine)
      Shell: 5.7.1 - /usr/bin/zsh
    Binaries:
      Node: 13.6.0 - ~/.nvm/versions/node/v13.6.0/bin/node
      npm: 6.14.4 - ~/.nvm/versions/node/v13.6.0/bin/npm
    npmPackages:
      expo: ^37.0.3 => 37.0.3 
      react: 16.9.0 => 16.9.0 
      react-native: https://github.com/expo/react-native/archive/sdk-37.0.0.tar.gz => 0.61.4 
    npmGlobalPackages:
      expo-cli: 3.17.5

Target: Android

### Steps to Reproduce

After updating the app with expo update to the new SDK37 the app starts with this error. I removed node_modules, cleaned the cache, and tried to update @react-navigation/stack too, but nothing seems to change.

I saw web support for react navigation is not guaranteed, but I'm targeting mobile currently.

### Expected Behavior

It should start correctly

### Actual Behavior

App failing to start with following error screen
![image](https://user-images.githubusercontent.com/18316027/78147416-c2e28280-7433-11ea-92d7-761c52bb778c.png)

### Reproducible Demo

It's hard to me to provide a demo.

**Comment by FutureOfRussia** [comment]
@xn3cr0nx Hi, I ran into the same problem. As I understand it, expo `start -c` don't cleared cache, `shift + r` helped me.

**Comment by brentvatne** [comment]
you might want to delete your lockfile along with node_modules and try again. i suspect this commit https://github.com/react-navigation/react-navigation/commit/9bb5cfded3f805797cf759386bb3bfe34d2e01e7 is maybe related, and some package might be out of sync

**Comment by brentvatne** [comment]
perhaps related https://github.com/expo/expo-cli/issues/1357

**Comment by xn3cr0nx** [comment]
@brentvatne I removed manually package-lock file too before opening the issue. That was not the issue. Maybe this is related to react-navigation, but since it appeared after upgrading to SDK37, I thought there was an error with expo. I think someone tried react navigation before delivering new SDK. 
Anyway I tried @FutureOfRussia solution and it worked. It seems there is a further cached somewhere (I remove ~/.cache/expo too before opening the issue). Didn't know about shift+r, but this did the trick. Thank you @FutureOfRussia
